packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -72,6 +72,18 @@ function getContentBlocks( blocks, isContentBlock ) {
 	return result;
 }
 
+function getShowTabs( blockName, tabSettings = {} ) {
+	if ( tabSettings[ blockName ] !== undefined ) {
+		return tabSettings[ blockName ];
+	}
+
+	if ( tabSettings.default !== undefined ) {
+		return tabSettings.default;
+	}
+
+	return window?.__experimentalEnableBlockInspectorTabs;
```

Comment:
I wonder if this should be the first thing checked in the function? At the moment the setting can bypass the experiment. The experiment should maybe overrule everything else.

Reply:
✅ Done. https://github.com/WordPress/gutenberg/pull/46321/commits/e5dd52f58bbf8de31645fbff93d4f85aedbf56d0